packages/discord.js/src/util/Channels.js

```diff
@@ -65,13 +66,85 @@ function createChannel(client, data, guild, { allowUnknownGuild, fromInteraction
         case ChannelType.GuildDirectory:
           channel = new (getDirectoryChannel())(guild, data, client);
           break;
+        case ChannelType.GuildForum:
+          channel = new (getForumChannel())(guild, data, client);
+          break;
       }
       if (channel && !allowUnknownGuild) guild.channels?.cache.set(channel.id, channel);
     }
   }
   return channel;
 }
 
+/**
+ * Transforms an API guild forum tag to camel-cased guild forum tag.
+ * @param {APIGuildForumTag} tag The tag to transform
+ * @returns {GuildForumTag}
+ * @ignore
+ */
+function transformAPIGuildForumTag(tag) {
+  return {
+    id: tag.id,
+    name: tag.name,
+    moderated: tag.moderated,
+    emoji:
+      tag.emoji_id ?? tag.emoji_name
```

Comment:
If prettier or eslint doesn't complain about this then I got my order of operations wrong but this has bitten my booty recently
```suggestion
      (tag.emoji_id ?? tag.emoji_name)
```

Reply:
Tests pass on this pull request though?